src/Symfony/Bundle/FrameworkBundle/Secrets/SodiumVault.php

```diff
@@ -100,7 +101,8 @@ public function reveal(string $name): ?string
         $this->lastMessage = null;
         $this->validateName($name);
 
-        if (!is_file($file = $this->pathPrefix.$name.'.'.substr_replace(md5($name), '.php', -26))) {
+        $filename = $this->getFilename($name);
```

Comment:
let's use $file everywhere as it was the case, this will prevent needles cs diff and fix L123/125 below